public/components/common/search/autocomplete.tsx

```diff
@@ -0,0 +1,270 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+import './search.scss';
+import React, { createElement, Fragment, useEffect, useRef, useState } from 'react';
+
+import { autocomplete } from '@algolia/autocomplete-js';
+import { RAW_QUERY } from '../../../common/constants/explorer';
+import { IQueryBarProps } from './search';
+import { handlePplRequest } from '../../../requests/ppl';
+
+// Possible suggestions (hardcoded)
+const firstCommand = [{ label: 'search' }, { label: 'source' }];
```

Comment:
i think i've mostly seen PPL starting with `source`, is there a use case to have it starts with `search`?